lib/logstash/inputs/syslog.rb

```diff
@@ -118,6 +119,7 @@ def server(protocol, output_queue)
   rescue => e
     if !stop?
       @logger.warn("syslog listener died", :protocol => protocol, :address => "#{@host}:#{@port}", :exception => e, :backtrace => e.backtrace)
+      @metric_errors.increment(:listener_exception)
```

Comment:
@ph @jordansissel I think now there are 3 places with similar discussions, well the same. I created https://github.com/elastic/logstash/issues/5716 to centralize, can we agree on pattern there? that would be awesome! then I will make sure to follow it everywhere 😄